src/ext/conversion/importProperties.ts

```diff
@@ -1,13 +1,49 @@
 import conversion from 'qlik-object-conversion';
+import { setValue } from 'qlik-chart-modules';
 
 import data from '../../qae/data';
-import { ExportFormat, PropTree } from '../../types';
+import { ExportFormat, PropTree, DimensionProperties, MeasureProperties } from '../../types';
+import { ColumnWidthTypes } from '../../table/constants';
+
+export const getColumnInfo = (
+  columnInfo: DimensionProperties[] | MeasureProperties[],
+  colIdx: number,
+  columnWidths: unknown,
```

Comment:
I think you should be able to type this, isn't it always a number[] or undefined?

Reply:
empty array, [] as well?

Reply:
not sure, but that would be covered by number[]

Reply:
right, then it is fine to type as `columnWidths?: number[],`

Reply:
Fixed